src/System.Management.Automation/engine/parser/tokenizer.cs

```diff
@@ -1229,23 +1229,125 @@ private int SkipBlockComment(int i)
             return i;
         }
 
-        private static char Backtick(char c)
+        private string Backtick(char c)
```

Comment:
That's not a bad idea.  Thanks!  The code at each of the six call sites would still need to add the result of calling ScanUnicodeEscapeSequence to the local sb/formatSb instances but the intent of that code would be clearer than the current string length > 1 checks.  I'll try this change later tonight.